source/features/forked-to.tsx

```diff
@@ -42,7 +67,28 @@ function createLink(baseRepo: string): string {
 		branch: 'HEAD'
```

Comment:
Remove this line here
```suggestion
```

Reply:
Or, actually, even better! This means that we can skip a lot of code.

I'll push something for you to have a look